daemons/fenced/fenced_remote.c

```diff
@@ -608,7 +608,7 @@ remote_op_timeout_one(gpointer userdata)
 
     crm_notice("Peer's '%s' action targeting %s for client %s timed out " CRM_XS
                " id=%.8s", op->action, op->target, op->client_name, op->id);
-    call_remote_stonith(op, NULL, pcmk_ok);
+    call_remote_stonith(op, NULL, -ETIME);
```

Comment:
In my tests, this value winds up not mattering, because it is not the final result that gets reported -- either the overall operation succeeds with a later attempt, or it ends up in request_peer_fencing()'s "No peers" block. However I suspect that there could be some code path that uses this as the final result, or could be if the code changes in the future, and it should be a failure. I definitely couldn't find any reason why setting it ok would be helpful.

Reply:
Didn't find a case where setting it ok would be helpful either.
Just couldn't wind my mind around it enough to assure it wouldn't change the behavior.
But yes - should be fine - and if not something else is wrong.